examples/default-values-backend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "backend_app_demo",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "scripts": {
+    "start": "yarn buildFrontend && tsc && node --insecure-http-parser built/src/index.js",
+    "buildFrontend": "parcel build public/index.html",
+    "setup": "NODE_OPTIONS=--insecure-http-parser ts-node src/setup.ts"
+  },
+  "author": "<john.whiles@contentful.com>",
```

Comment:
let's change this

Reply:
To something like just "Contentful" or "team+extensibility@contentful.com"?